aws-ts-ec2-provisioners/index.ts

```diff
@@ -0,0 +1,63 @@
+// Copyright 2016-2019, Pulumi Corporation.  All rights reserved.
+
+import * as aws from "@pulumi/aws";
+import * as pulumi from "@pulumi/pulumi";
+import { CopyFile, RemoteExec } from "./provisioners";
+
+// Get the config ready to go.
+const config = new pulumi.Config();
+const publicKey = config.require("publicKey");
+const privateKey = config.require("privateKey");
```

Comment:
Any reason not to use `requireSecret` here (and `getSecret`/`requireSecret` in the following line)?